contributing.md

```diff
@@ -40,3 +40,53 @@ You'll need a [GitHub account](https://github.com/join)!
+
+### PCB design
+
+It involves creating the physical layout of components and their interconnections on a PCB (Printed Circuit Board). It translates the logical and functional aspects of a circuit design into a tangible, manufacturable form. Key aspects of EDA with PCB design capabilities are:
+
+- Component Placement: Such as integrated circuits (ICs), resistors, capacitors, and connectors, onto the PCB.
+- Routing: involves creating the copper traces that connect the pins of different components on the PCB. EDAs for PCB design can verifi aspects such as signal integrity, impedance matching, minimizing electromagnetic interference (EMI), and avoiding signal crosstalk.
+- Design Rule Check (DRC): EDA tools perform design rule checks to ensure that the layout adheres to the manufacturer's specifications and constraints.7
+- Gerber Files: After the PCB layout is complete and verified, Gerber files are generated. These files contain the information needed for PCB fabrication, including copper layers, silkscreen, and solder mask.
+- 3D Visualization: Some advanced EDA tools offer 3D visualization features, allowing designers to view the PCB layout in a three-dimensional space.
+
+## Tier description
+
+### ${\color{red}S}$
+
+Full-featured and professional-level EDAs, widely adopted in the industry for highly complex electronic projects. Highly active project, whether proprietary or open source.
+
+### ${\color{orange}A}$
+
+Very popular, with a large community, and with a very active and accelerated software development. It contains advanced features but not so widely adopted in the industry as it lacks cutting-edge features. Even though, it can be adopted to complex electronic systems, usually as a free or OS-compatible alternative to the ${\color{red}S}$ tier.
+
+### ${\color{yellow}B}$
+
+A EDA with a small community yet and not so popular, but with a promising and very active software development. Software in this tier tends to grwo into tier ${\color{orange}A}$ as they get matured. They can have some nice features for more complex electronic projects, but they are usually more limited when compared to tier ${\color{orange}A}$ or ${\color{red}S}$.
+
+### ${\color{green}C}$
+
+Single-featured EDAs with a small community. It lacks more complex functionalities, and the software development is not very active. Additionally, it is usually used in a niche. EDAs in this tier are not expected to grow much further than they already are. Even though, the software can fully fullfil its taks and can be used for small- and medium-sized projects.
+
+### ${\color{light blue}D}$
+
+Old and single-featured EDAs with a rarely active software development. EDAs in this tier tend to become obsolete throughout the time.
+
+### ${\color{purple}E}$
+
+Abandoned EDA projects with obsolete features. These EDAs are listed for historical reasons, as they were relevant in the past.
```

Comment:
I just created a tier list as suggested by @joaoantoniocardoso . Let me know if this is any good.

Reply:
Looks great! Thank you! =D